keras/preprocessing/sequence.py

```diff
@@ -4,31 +4,39 @@
 import random
 from six.moves import range
 
-def pad_sequences(sequences, maxlen=None, dtype='int32', padding='pre'):
+
+def pad_sequences(sequences, maxlen=None, dtype='int32', padding='pre', vector_sequence=False):
```

Comment:
`vector_sequence=False` does not seem necessary, you could just check whether multidimensional arrays are being passed in `sequences`.